virt/virtual_machines/advanced_vm_management/virt-vm-control-plane-tuning.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="virt-vm-control-plane-tuning"]
+= Virtual machine control plane tuning
+
+include::_attributes/common-attributes.adoc[]
+:context: virt-control-plane-tuning
+
+OpenShift Virtualization offers several tuning options at the control-plane level. This includes:
```

Comment:
There are only two options here so I would rewrite this to avoid using "several."

```suggestion
OpenShift Virtualization offers the following tuning options at the control-plane level:
```

Reply:
(If you keep "this includes" it should be "these include" because options is plural)